core/config/project_settings.cpp

```diff
@@ -66,6 +68,73 @@ String ProjectSettings::get_imported_files_path() const {
 	return get_project_data_path().plus_file("imported");
 }
 
+// Returns the features that a project must have when opened with this build of Godot.
+// This is used by the project manager to provide the initial_settings for config/features.
+const PackedStringArray ProjectSettings::get_required_features() {
+	PackedStringArray features = PackedStringArray();
+	features.append(VERSION_BRANCH);
+#ifdef REAL_T_IS_DOUBLE
+	features.append("Double-precision");
```

Comment:
Would need to settle on a format, "Double-precision" and "Vulkan Clustered" are not consistent.

Reply:
I went with "Double Precision". Also I removed "Single-precision" since reduz said it wasn't necessary as a warning, which is mostly true.